### PR TITLE
MAUI-3111 : Prepare migration document for progress bar control.

### PR DESCRIPTION
**Pull request by BalachandarSelvam**
Hi @vigneshrameshsync ,

I have prepared migration document for the maui progress bar control. please review it.

Regards,
Balachandar.

### CODE REVIEW

File: MAUI/ProgressBar/migration.md [hunk 1]
```diff
@@ -0,0 +1,587 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI  ProgressBar | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin SfProgressBar to Syncfusion .NET MAUI SfProgressBar control and more.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:4]: description: Learn here all about Migrating from the Syncfusion Xamarin SfProgressBar to the Syncfusion .NET MAUI SfProgressBar control and more.

File: MAUI/ProgressBar/migration.md [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfProgressBar to .NET MAUI SfProgressBar
+
+To make the migration from the [Xamarin SfProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfProgressBar` easier, most of the APIs from the Xamarin SfProgressBar were kept in the.NET MAUI SfProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfProgressBar, some of the APIs have been renamed. Please find the difference in the following topics.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:12]: To make the migration from the [Xamarin SfProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfProgressBar` easier, most of the APIs from the Xamarin SfProgressBar were kept in the.NET MAUI SfProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfProgressBar, some of the APIs have been renamed. Please find the difference in the following topics:

File: MAUI/ProgressBar/migration.md [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+
+## Initialize control
+
+To initialize the control, import the ProgressBar namespace and initialize SfProgressBar as shown in the following code sample.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:16]: To initialize the control, import the ProgressBar namespace and initialize the SfProgressBar as shown in the following code sample:

File: MAUI/ProgressBar/migration.md [hunk 4]
```diff
@@ -0,0 +1,587 @@
...
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[EasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_EasingEffect)'| markdownify }}</td>
+<td>{{'AnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for progress animation.</td></tr>
+<tr>
+<td>{{'[GapWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_GapWidth)'| markdownify }}</td>
+<td>{{'SegmentGapWidth'| markdownify }}</td>
+<td>Gets or sets the value that determine the gap between the segments.</td></tr>
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:106]: <td>Gets or sets the value that determines the gap between the segments.</td></tr>

File: MAUI/ProgressBar/migration.md [hunk 5]
```diff
@@ -0,0 +1,587 @@
...
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[SecondaryProgressColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryProgressColor)'| markdownify }}</td>
+<td>{{'SecondaryProgressFill'| markdownify }}</td>
+<td>Gets or sets the brush that paints the interior area of the secondary progress.</td></tr>
+<tr>
+<td>{{'[CornerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_CornerRadius)'| markdownify }}</td>
+<td>Divided into {{'TrackCornerRadius'| markdownify }}, {{'ProgressCornerRadius'| markdownify }} and {{'SecondaryProgressCornerRadius'| markdownify }}</td>
+<td>Gets or sets a value to determine the corner radius of the track, progress and secondary progress.</td></tr>
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:353]: <td>Gets or sets a value to determine the corner radius of the track, progress, and secondary progress.</td></tr>

File: MAUI/ProgressBar/migration.md [hunk 6]
```diff
@@ -0,0 +1,587 @@
...
+
+<table> 
+<tr>
+<th>Xamarin SfProgressBar</th>
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[IndicatorInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorInnerRadius)'| markdownify }}</td>
+<td>{{'ProgressThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness for the progress. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:454]: <td>Gets or sets a value that specifies the thickness of the progress. You can specify the value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>

File: MAUI/ProgressBar/migration.md [hunk 7]
```diff
@@ -0,0 +1,587 @@
...
+<tr>
+<td>{{'[IndicatorOuterRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorOuterRadius)'| markdownify }}</td>
+<td>{{'ProgressRadiusFactor'| markdownify }}</td>
+<td>Gets or sets a value that specifies the outer radius factor of the progress.</td></tr>
+<tr>
+<td>{{'[ShowProgressValue](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue)'| markdownify }}</td>
+<td>{{'-'| markdownify }}</td>
+<td>Gets or sets a value indicating whether to show progress value label or not in SfCircularProgressBar.</td></tr>
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:462]: <td>Gets or sets a value indicating whether to show the progress value label or not in SfCircularProgressBar.</td></tr>

File: MAUI/ProgressBar/migration.md [hunk 8]
```diff
@@ -0,0 +1,587 @@
...
+<tr>
+<td>{{'[TrackInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_TrackInnerRadius)'| markdownify }}</td>
+<td>{{'TrackThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness of track in circular progress bar. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:466]: <td>Gets or sets a value that specifies the thickness of the track in the circular progress bar. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>

File: MAUI/ProgressBar/migration.md [hunk 9]
```diff
@@ -0,0 +1,587 @@
...
+<td>{{'ProgressCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the progress.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'TrackCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the track.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'ThicknessUnit'| markdownify }}</td>
+<td>Gets or sets enum value that indicates to calculate the track and progress thickness either in logical pixel or radius factor.</td></tr>
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:482]: <td>Gets or sets the enum value that indicates to calculate the track and progress thickness either in logical pixel or radius factor.</td></tr>

File: MAUI/ProgressBar/migration.md [hunk 10]
```diff
@@ -0,0 +1,587 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+## Unsupported features from Xamarin.Forms
+
+* [`ShowProgressValue`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue) support has not been provided in the SfCircularProgressBar. Instead, you can achieve it using the `Content` property. Please visit Custom Content page for more details about the `Content` property.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:585]: * [`ShowProgressValue`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue) support has not been provided in the SfCircularProgressBar. Instead, you can achieve it using the `Content` property. Please visit the Custom Content page for more details about the `Content` property.

Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:585]: done

File: MAUI/ProgressBar/migration.md (hunk 9 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:482]: done

File: MAUI/ProgressBar/migration.md (hunk 8 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:466]: done

File: MAUI/ProgressBar/migration.md (hunk 7 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:462]: done

File: MAUI/ProgressBar/migration.md (hunk 4 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:106]: done

File: MAUI/ProgressBar/migration.md (hunk 1 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:4]: Done

File: MAUI/ProgressBar/migration.md [hunk 11]
```diff
@@ -0,0 +1,725 @@
...
+<td>Gets or sets a value that specifies the easing effect for progress animation.</td></tr>
+<tr>
+<td>{{'[IndeterminateEasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateEasingEffect)'| markdownify }}</td>
+<td>{{'IndeterminateAnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for indeterminate animation.</td></tr>
+<tr>
+<td>{{'[IndeterminateIndicatorWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateIndicatorWidth)'| markdownify }}</td>
+<td>{{'IndeterminateIndicatorWidthFactor'| markdownify }}</td>
+<td>Gets or sets the value that specifies width of the indeterminate indicator.</td></tr>
+</table>
```
Comment (vigneshrameshsync) [inline MAUI/ProgressBar/migration.md:155]: events are missing.

File: MAUI/ProgressBar/migration.md [hunk 12]
```diff
@@ -0,0 +1,725 @@
...
+<td>{{'[IndeterminateEasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateEasingEffect)'| markdownify }}</td>
+<td>{{'IndeterminateAnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for indeterminate animation.</td></tr>
+<tr>
+<td>{{'[IndeterminateIndicatorWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateIndicatorWidth)'| markdownify }}</td>
+<td>{{'IndeterminateIndicatorWidthFactor'| markdownify }}</td>
+<td>Gets or sets the value that specifies width of the indeterminate indicator.</td></tr>
+</table> 
+
+<table>
```
Comment (vigneshrameshsync) [inline MAUI/ProgressBar/migration.md:157]: Before adding the code snippet, provide the below sentence

The following code example explains how to initialize the Xamarin progress bar and the .NET MAUI progress bar along with common properties.

File: MAUI/ProgressBar/migration.md [hunk 13]
```diff
@@ -0,0 +1,725 @@
...
+title: Migrate from Xamarin to .NET MAUI  ProgressBar | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin SfProgressBar to Syncfusion .NET MAUI SfProgressBar control and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfProgressBar to .NET MAUI SfProgressBar
+
+To make the migration from the [Xamarin SfProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfProgressBar` easier, most of the APIs from the Xamarin SfProgressBar were kept in the .NET MAUI SfProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfProgressBar, some of the APIs have been renamed. Please find the difference in the following topics.
```
Comment (vigneshrameshsync) [inline MAUI/ProgressBar/migration.md:12]: SfProgressBar -> ProgressBar

File: MAUI/ProgressBar/migration.md [hunk 14]
```diff
@@ -0,0 +1,725 @@
...
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+
+## SfLinearProgressBar
+
+<table>
```
Comment (vigneshrameshsync) [inline MAUI/ProgressBar/migration.md:412]: Before adding the code snippet, provide the below sentence

The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfCircularProgressBar along with its properties.

File: MAUI/ProgressBar/migration.md [hunk 15]
```diff
@@ -0,0 +1,725 @@
...
+<td>Divided into {{'TrackCornerRadius'| markdownify }}, {{'ProgressCornerRadius'| markdownify }} and {{'SecondaryProgressCornerRadius'| markdownify }}</td>
+<td>Gets or sets a value to determine the corner radius of the track, progress, and secondary progress.</td></tr>
+<tr>
+<td>{{'[SecondaryAnimationDuration](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryAnimationDuration)'| markdownify }}</td>
+<td>{{'SecondaryAnimationDuration'| markdownify }}</td>
+<td>Gets or sets a value that specifies the secondary progress animation duration in milliseconds.</td></tr>
+
+</table> 
+
+<table>
```
Comment (vigneshrameshsync) [inline MAUI/ProgressBar/migration.md:440]: Before adding the code snippet, provide the below sentence

The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfCircularProgressBar along with its properties.

File: MAUI/ProgressBar/migration.md [hunk 16]
```diff
@@ -0,0 +1,725 @@
...
+<td>{{'TrackCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the track.</td></tr>
+<tr>
+<td>{{'[ShowProgressValue](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue)'| markdownify }}</td>
+<td>{{'-'| markdownify }}</td>
+<td>Gets or sets a value indicating whether to show the progress value label or not in SfCircularProgressBar.</td></tr>
+
+</table> 
+
+<table>
```
Comment (vigneshrameshsync) [inline MAUI/ProgressBar/migration.md:594]: Before adding the code snippet, provide the below sentence

The following code example explains how to initialize the Xamarin SfCircularProgressBar and the .NET MAUI SfCircularProgressBar along with its properties.

Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:600]: done

File: MAUI/ProgressBar/migration.md (hunk 15 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:444]: done

File: MAUI/ProgressBar/migration.md (hunk 13 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md]: done

File: MAUI/ProgressBar/migration.md (hunk 12 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:159]: done

File: MAUI/ProgressBar/migration.md (hunk 11 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md:155]: done

File: MAUI/ProgressBar/migration.md [hunk 17]
```diff
@@ -0,0 +1,893 @@
...
+<td>{{'[IndeterminateEasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateEasingEffect)'| markdownify }}</td>
+<td>{{'IndeterminateAnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for indeterminate animation.</td></tr>
+<tr>
+<td>{{'[IndeterminateIndicatorWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_IndeterminateIndicatorWidth)'| markdownify }}</td>
+<td>{{'IndeterminateIndicatorWidthFactor'| markdownify }}</td>
+<td>Gets or sets the value that specifies width of the indeterminate indicator.</td></tr>
+</table> 
+
+The following code example explains how to use the common properties in the Xamarin progress bar and the .NET MAUI progress bar.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:157]: The following code example explains how to use the common properties in the Xamarin progress bar and the .NET MAUI progress bar:

File: MAUI/ProgressBar/migration.md [hunk 18]
```diff
@@ -0,0 +1,893 @@
...
+<td>Divided into {{'TrackCornerRadius'| markdownify }}, {{'ProgressCornerRadius'| markdownify }} and {{'SecondaryProgressCornerRadius'| markdownify }}</td>
+<td>Gets or sets a value to determine the corner radius of the track, progress, and secondary progress.</td></tr>
+<tr>
+<td>{{'[SecondaryAnimationDuration](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryAnimationDuration)'| markdownify }}</td>
+<td>{{'SecondaryAnimationDuration'| markdownify }}</td>
+<td>Gets or sets a value that specifies the secondary progress animation duration in milliseconds.</td></tr>
+
+</table> 
+
+The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfLinearProgressBar along with its properties.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:442]: The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfLinearProgressBar along with its properties:

File: MAUI/ProgressBar/migration.md [hunk 19]
```diff
@@ -0,0 +1,893 @@
...
+<td>{{'TrackCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the track.</td></tr>
+<tr>
+<td>{{'[ShowProgressValue](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue)'| markdownify }}</td>
+<td>{{'-'| markdownify }}</td>
+<td>Gets or sets a value indicating whether to show the progress value label or not in SfCircularProgressBar.</td></tr>
+
+</table> 
+
+The following code example explains how to initialize the Xamarin SfCircularProgressBar and the .NET MAUI SfCircularProgressBar along with its properties.
```
Comment (OnyimbiNelson) [inline MAUI/ProgressBar/migration.md:598]: The following code example explains how to initialize the Xamarin SfCircularProgressBar and the .NET MAUI SfCircularProgressBar along with its properties:

File: MAUI/ProgressBar/migration.md (hunk 2 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md]: not included colon

File: MAUI/ProgressBar/migration.md (hunk 3 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md]: not included colon

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

File: MAUI/ProgressBar/migration.md (hunk 5 above)
Comment (BalachandarSelvam) [reply MAUI/ProgressBar/migration.md]: done

**Pull request merged**